_redirects

```diff
@@ -0,0 +1,2 @@
+http://developer.helium.com/* https://docs.helium.com
```

Comment:
I don't think this is quite right?

I think it needs to be:

`https://developer.helium.com/* https://docs.helium.com/:splat 301!`

Note the `https` and the splat/301 usage